.gitignore

```diff
@@ -11,11 +11,11 @@ a.out
 /doc/doctrees/
 /doc/html/
 /doc/man/*.1
-/libexec/charliecloud
-/libexec/contributors.bash
-/libexec/version.py
-/libexec/version.sh
-/libexec/version.txt
+/lib/charliecloud
+/lib/contributors.bash
+/lib/version.py
+/lib/version.sh
+/lib/version.txt
```

Comment:
I'm not sure I like putting these generically named files directly into`/lib`. What are your thoughts about putting these files in a charliecloud sub directory, for example`/lib/charliecloud`, as the default behavior?